boards/xtensa/qemu_xtensa/doc/board.rst

```diff
@@ -0,0 +1,45 @@
+.. _qemu_xtensa:
+
+Xtensa Emulation (QEMU)
+#######################
+
+Overview
+********
+
+The QEMU board configuration is used to emulate the Xtensa architecture. This board
+configuration provides support for the Xtensa simulation environment.
+
+Hardware
+********
+
+Programming and Debugging
+*************************
+
+Use this board configuration to run basic Zephyr applications and kernel tests
```

Comment:
Use the `BOARD=qemu_xtensa` configuration to run basic Zephyr applications
     and kernel tests in the QEMU emulated environment, for example, with the
     :ref:`synchronization_sample`: